system-int-test/test/cmr/system_int_test/search/autocomplete_collection_facet_search_test.clj

```diff
@@ -35,7 +39,6 @@
     (doseq [doc documents] (debug "ingested " doc))
     (index/wait-until-indexed)
     (f)
-    (doseq [doc documents] (esd/delete conn "1_autocomplete" "suggestion" (:_id doc)))
     (index/wait-until-indexed)))
```

Comment:
can remove this if we are no longer cleaning up the created documents. Can tests be run consecutively without this cleanup happening i.e. the counts remain correct?